astropy/coordinates/builtin_frames/cirs_observed_transforms.py

```diff
@@ -40,7 +40,8 @@ def cirs_to_altaz(cirs_coo, altaz_frame):
     xp, yp = get_polar_motion(obstime)
 
     #first set up the astrometry context for CIRS<->AltAz
-    astrom = erfa.apio13(obstime.jd1, obstime.jd2,
+    obstime_utc = obstime if obstime.scale == 'utc' else obstime.utc
```

Comment:
@eteq @mhvk - I've created issue #4309 to discuss this since it's getting a bit OT here.